Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -433,7 +435,7 @@ public class ModuleDefinition : RangeNode, IDeclarationOrUsage, IAttributeBearin
     return true;
   }
 
-  public void ProcessPrefixNamedModules() {
+  public void ProcessPrefixNamedModules(bool setRange) {
```

Comment:
Do you actually use this argument anywhere?

Reply:
It's not used. Sorry, I should have removed it already.